LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Microsoft Corporation
+Copyright (c) present Microsoft Corporation
```

Comment:
please remove "present" and it should be sufficient. Thanks!

Reply:
Is the reason [here](https://stackoverflow.com/a/2391555/1658314)